src/Features/Core/Portable/InlineMethod/AbstractInlineMethodRefactoringProvider.cs

```diff
@@ -202,6 +200,14 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 }
             }
 
+            var callerMethodSymbol = GetContainingMethodSymbol(calleeMethodInvocationNode, semanticModel, cancellationToken);
+            var callerMethodReferences = symbolDeclarationService.GetDeclarations(callerMethodSymbol);
+            if (callerMethodReferences.Length != 1)
+            {
+                return;
+            }
+
+            var callerMethodNode = await callerMethodReferences[0].GetSyntaxAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
can you add a test for both "caller is in a partial method" and "callee is a partial method".

Reply:
Added